setup.py

```diff
@@ -37,11 +37,14 @@
 setup_requires = ["pytest-runner>=2.6.2"]
 
 install_requires = [
-    "github3.py>=0.9.3",
     "click>=7.0",
-    "PyYAML>=3.13",
-    "python-dotenv>=0.10.3",
+    "datetime>=4.3",
+    "github3.py>=0.9.3",
     "lazy-load>=0.8.2",
+    "python-dotenv>=0.10.3",
+    "pytz>=2019.2",
+    "PyYAML>=3.13",
+    "setuptools>=41.0.1",
```

Comment:
This shouldn't be needed actually. It's usually installed already in your system or e.g. [updated manually for travis](https://github.com/inveniosoftware/autobot/blob/7b0f01d7448300e1c2283ffc8973b65099948183/.travis.yml#L35)